readme.md

```diff
@@ -446,6 +446,7 @@
 - [Vehicle Security and Car Hacking](https://github.com/jaredthecoder/awesome-vehicle-security#readme)
 - [Web Security](https://github.com/qazbnm456/awesome-web-security#readme) - Security of web apps & services.
 - [Lockpicking](https://github.com/meitar/awesome-lockpicking#readme) - The art of unlocking a lock by manipulating its components without the key.
+- [Fuzzing](https://github.com/cpuu/awesome-fuzzing#readme) - Fuzzing for security testing.
```

Comment:
> The entry in the Awesome list should:
* Include a short description about the project/theme of the list. **It should not describe the list itself.**